src/components/SearchSuggestions.vue

```diff
@@ -1,7 +1,12 @@
 <template>
     <div class="suggestions-container absolute">
         <ul>
-            <li v-for="(suggestion, i) in searchSuggestions" :key="i" @mouseover="onMouseOver(i)">
+            <li
+                v-for="(suggestion, i) in searchSuggestions"
+                :key="i"
+                @mouseover="onMouseOver(i)"
+                @mousedown="onClick(i)"
```

Comment:
```suggestion
                @click="onClick(i)"
```
That's probably the more common way to do this?

Reply:
I've changed the code according to your suggestions. (Note: A quick `grep` showed that Piped never uses `@mousedown`, only `@click`. :wink: )